.angular-cli.json

```diff
@@ -19,7 +19,8 @@
       "testTsconfig": "tsconfig.spec.json",
       "prefix": "app",
       "styles": [
-        "styles.css"
+        "styles.css",
+        "custom.scss"
```

Comment:
you can use only one CSS entry file if you want - simple: rename `styles.css` to `styles.scss` so it can act as a one css styles entry point